tfx/components/tuner/component.py

```diff
@@ -84,9 +84,11 @@ def tuner_fn(fn_args: FnArgs) -> TunerFnResult:
         'module_file' for the required signature of the UDF. Exactly one of
         'module_file' or 'tuner_fn' must be supplied.
       train_args: A trainer_pb2.TrainArgs instance, containing args used for
-        training. Current only num_steps is available.
+        training. Current only splits and num_steps are available. Default
+        behavior (when splits is empty) is train on 'train' splits.
```

Comment:
... is train on 'train' split.

("split" singular)

Reply:
got it, thanks for the catch, so detailed!